json_document/tests/test_fragment_class.py

```diff
@@ -0,0 +1,130 @@
+# This file is part of json-document
+#
+# json-document is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation
+#
+# json-document is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with json-document.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for fragment classes."""
+
+from unittest2 import TestCase
+
+from json_document.document import DocumentFragment
+
+
+class DocumentFragmentClassTests(TestCase):
+    """Tests related to fragment classes."""
```

Comment:
That depends on the flavour.  There are two conflicting error codes, due to the pep 257 changing.

    D203: 1 blank line required before class docstring

    D211: No blank lines allowed before class docstring

If we want to follow the current pep, we would need to disable D203, and then remove blank lines before the class docstring.

However if you prefer the old pep, I am more than happy to add a blank line.  My other projects usually take this approach, as I find that extra blank line helpful.